Resources/translations/SonataDashboardBundle.de.xliff

```diff
@@ -197,7 +197,23 @@
             <trans-unit id="48" resname="compose.status_error">
                 <source>compose.status_error</source>
                 <target>Es ist ein Fehler beim Speichern des Status aufgetreten.</target>
-            </trans-unit>  
+            </trans-unit>
+            <trans-unit id="49" resname="form.label_code">
+                <source>form.label_code</source>
+                <target>Code</target>
+            </trans-unit>
+            <trans-unit id="50" resname="form.label_layout">
+                <source>form.label_layout</source>
+                <target>Layout</target>
+            </trans-unit>
+            <trans-unit id="51" resname="form.label_class">
+                <source>form.label_class</source>
+                <target>Class</target>
```

Comment:
The ``class`` label is for *css classes*. Maybe **CSS Class** or **CSS Klasse**?